.gitpod.yml

```diff
@@ -0,0 +1,14 @@
+image: nfcore/gitpod:latest
+
+vscode:
+  extensions: # based on nf-core.nf-core-extensionpack
+    - codezombiech.gitignore # Language support for .gitignore files
+    # - cssho.vscode-svgviewer              # SVG viewer
+    - davidanson.vscode-markdownlint # Markdown/CommonMark linting and style checking for Visual Studio Code
```

Comment:
There was a change in this line also in the last template update

Reply:
Adjusted this to the latest update